spec/requests/v1/transactions/create_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
```

Comment:
@yunixon move to file `spec/requests/api/v1/transactions/create_spec.rb`. Requests specs may be not only for api.